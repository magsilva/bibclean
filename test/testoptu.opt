-fix-math -max-width 0

