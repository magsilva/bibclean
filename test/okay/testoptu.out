@Misc{Blow:2013:Ea,
  author =       "Joe Blow",
  title =        "{O($^3$P) $+$ CH$_3$SH} reactions",
  expect =       "{O($^3$P) $+$ CH$_3$SH} reactions",
  year =         "2013",
}

@Misc{Blow:2013:Eb,
  author =       "Joe Blow",
  title =        "The {ENIAC}'s 1949 Determination of $ \pi $",
  expect =       "The {ENIAC}'s 1949 Determination of $ \pi $",
  year =         "2013",
}

@Misc{Blow:2013:Ec,
  author =       "Joe Blow",
  title =        "A Course in the Geometry of $n$-Dimensions",
  expect =       "A Course in the Geometry of $n$-Dimensions",
  year =         "2013",
}

@Misc{Blow:2013:Ed,
  author =       "Joe Blow",
  title =        "Compression of {$3$D} Points",
  expect =       "Compression of {$3$D} Points",
  year =         "2013",
}

@Misc{Blow:2013:Ee,
  author =       "Joe Blow",
  title =        "{P$^2$SNR}: Perceptual Full-Reference Image Quality Assessment for {JPEG2000}",
  expect =       "{P$^2$SNR}: Perceptual Full-Reference Image Quality Assessment for {JPEG2000}",
  year =         "2013",
}

@Misc{Blow:2013:Ef,
  author =       "Joe Blow",
  title =        "The \$10 bet",
  expect =       "The \$10 bet",
  year =         "2013",
}

@Misc{Blow:2013:Eg,
  author =       "Joe Blow",
  title =        "{$ P_0 $}-Matrix Products of Matrices",
  expect =       "{$ P_0 $}-Matrix Products of Matrices",
  year =         "2013",
}

@Misc{Blow:2013:Eh,
  author =       "Joe Blow",
  title =        "Atomic Intentions in {Jason$^+$}",
  expect =       "Atomic Intentions in {Jason$^+$}",
  year =         "2013",
}

@Misc{Blow:2013:Ei,
  author =       "Joe Blow",
  title =        "{GENote} v. $ \beta $: a {Web} Tool Prototype",
  expect =       "{GENote} v. $ \beta $: a {Web} Tool Prototype",
  year =         "2013",
}

@Misc{Blow:2013:Ej,
  author =       "Joe Blow",
  title =        "{Polyakov} Action on {$ (\rho, G) $}-Functions",
  expect =       "{Polyakov} Action on {$ (\rho, G) $}-Functions",
  year =         "2013",
}

@Misc{Blow:2013:Ek,
  author =       "Joe Blow",
  title =        "A New Class of $ \epsilon $-Optimal Learning Automata",
  expect =       "A New Class of $ \epsilon $-Optimal Learning Automata",
  year =         "2013",
}

@Misc{Blow:2013:El,
  author =       "Joe Blow",
  title =        "Example of {$R$} Functions Management",
  expect =       "Example of {$R$} Functions Management",
  year =         "2013",
}

@Misc{Blow:2013:Em,
  author =       "Joe Blow",
  title =        "{$ H_\infty $} Output Tracking Control",
  expect =       "{$ H_\infty $} Output Tracking Control",
  year =         "2013",
}

@Misc{Blow:2013:En,
  author =       "Joe Blow",
  title =        "Ameliorating {$ {\rm GM}(1, 1) $} Model",
  expect =       "Ameliorating {$ {\rm GM}(1, 1) $} Model",
  year =         "2013",
}

@Misc{Blow:2013:Eo,
  author =       "Joe Blow",
  title =        "$ \sqrt {3}$-Based 1-Form Subdivision",
  expect =       "$ \sqrt {3} $-Based 1-Form Subdivision",
  year =         "2013",
}

@Misc{Blow:2013:Ep,
  author =       "Joe Blow",
  title =        "Symmetric $ \alpha $-stable distributions for $ q > 1$",
  expect =       "Symmetric $ \alpha $-stable distributions for $ q > 1 $",
  year =         "2013",
}

@Misc{Blow:2013:Eq,
  author =       "Joe Blow",
  title =        "{Dirac} spin-$ 1 / 2 $ particles",
  expect =       "{Dirac} spin-$ 1 / 2 $ particles",
  year =         "2013",
}

@Misc{Blow:2013:Er,
  author =       "Joe Blow",
  title =        "The $ \beta \gamma $-system, and {$ V_4$} of {$ {\rm sl}(2, \mathbb {C})$}",
  expect =       "The $ \beta \gamma $-system, and {$ V_4 $} of {$ {\rm sl}(2, \mathbb {C}) $}",
  year =         "2013",
}

@Misc{Blow:2013:Es,
  author =       "Joe Blow",
  title =        "The {$ \Phi^{2k} $} theory",
  expect =       "The {$ \Phi^{2k} $} theory",
  year =         "2013",
}

@Misc{Blow:2013:Et,
  author =       "Joe Blow",
  title =        "The $ \chi^2 $ test",
  expect =       "The $ \chi^2 $ test",
  year =         "2013",
}

@Misc{Blow:2013:Eu,
  author =       "Joe Blow",
  title =        "The {$ A, D, E $} family",
  expect =       "The {$ A, D, E $} family",
  year =         "2013",
}

@Misc{Blow:2013:Ev,
  author =       "Joe Blow",
  title =        "{$ {\rm SU}(6) \supset {\rm SU}(3) \otimes {\rm SU}(2) $} and {$ {\rm SU}(8) \supset {\rm SU}(4) \otimes {\rm SU}(2) $} {Clebsch--Gordan} coefficients",
  expect =       "{$ {\rm SU}(6) \supset {\rm SU}(3) \otimes {\rm SU}(2) $} and {$ {\rm SU}(8) \supset {\rm SU}(4) \otimes {\rm SU}(2)$} {Clebsch--Gordan} coefficients",
  year =         "2013",
}

@Misc{Blow:2013:Ey,
  author =       "Joe Blow",
  title =        "Unconfined $ - a / r + b r^2 $ potentials in $d$-dimensions",
  expect =       "Unconfined $ - a / r + b r^2 $ potentials in $d$-dimensions",
  year =         "2013",
}

@Misc{Blow:2013:Ez,
  author =       "Joe Blow",
  title =        "{CaCuO$_2$} antiferromagnetism",
  expect =       "{CaCuO$_2$} antiferromagnetism",
  year =         "2013",
}

@Misc{Blow:2013:Eaa,
  author =       "Joe Blow",
  title =        "Theoretical studies of {Au$_m$} and {PtAu$_n$} clusters and their {N$_2$} and {O$_2$} adsorption complexes",
  expect =       "Theoretical studies of {Au$_m$} and {PtAu$_n$} clusters and their {N$_2$} and {O$_2$} adsorption complexes",
  year =         "2013",
}

@Misc{Blow:2013:Eab,
  author =       "Joe Blow",
  title =        "Systematic assessment on aqueous {pK$_a$} and {pK$_b$} of an amino acid base",
  expect =       "Systematic assessment on aqueous {pK$_a$} and {pK$_b$} of an amino acid base",
  year =         "2013",
}

@Misc{Blow:2013:Eac,
  author =       "Joe Blow",
  title =        "Density functional studies of {Li$_N$} and {Li$_N^+$} ({$ N = 2$}--{$ 30$}) clusters: {Structure}, binding and charge distribution",
  expect =       "Density functional studies of {Li$_N$} and {Li$_N^+$} ($ N = 2 $--$ 30 $) clusters: {Structure}, binding and charge distribution",
  year =         "2013",
}

@Misc{Blow:2013:Ead,
  author =       "Joe Blow",
  title =        "Calculation of the {H$_2^+$} rovibrational energies and spectroscopic constants in the $ 2 p \pi $, $ 3 d \sigma $, $ 4 d \sigma $, $ 4 f \pi $, $ 4 f \sigma $, $ 5 g \sigma $, and $ 6 i \sigma $ electronic states",
  expect =       "Calculation of the {H$_2^+$} rovibrational energies and spectroscopic constants in the $ 2 p \pi $, $ 3 d \sigma $, $ 4 d \sigma $, $ 4 f \pi $, $ 4 f \sigma $, $ 5 g \sigma $, and $ 6 i \sigma $ electronic states",
  year =         "2013",
}

@Misc{Blow:2013:Eae,
  author =       "Joe Blow",
  title =        "Complexes of rare gases with {H$_2$S} and {H$_2$S$_2$}",
  expect =       "Complexes of rare gases with {H$_2$S} and {H$_2$S$_2$}",
  year =         "2013",
}

@Misc{Blow:2013:Eaf,
  author =       "Joe Blow",
  title =        "Solvent effects on {S$_N$2} reaction",
  expect =       "Solvent effects on {S$_N$2} reaction",
  year =         "2013",
}

@Misc{Blow:2013:Eag,
  author =       "Joe Blow",
  title =        "Reactions of {$^1$S}, {$^1$D}, and {$^3$P} carbon",
  expect =       "Reactions of {$^1$S}, {$^1$D}, and {$^3$P} carbon",
  year =         "2013",
}

@Misc{Blow:2013:Eah,
  author =       "Joe Blow",
  title =        "Dispersion coefficients for {Li$^+$--H} and {Li$^+$--He} systems with {Coulomb} and screened {Coulomb} potentials",
  expect =       "Dispersion coefficients for {Li$^+$--H} and {Li$^+$--He} systems with {Coulomb} and screened {Coulomb} potentials",
  year =         "2013",
}
